adapters/rest-adapter-generator/src/main/java/uk/gov/justice/services/adapters/rest/generator/Generators.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.services.adapters.rest.generator;
+
+import static uk.gov.justice.services.adapters.rest.generator.Names.baseUriPathWithoutContext;
+
+import uk.gov.justice.services.core.annotation.Component;
+
+import java.util.Comparator;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.raml.model.MimeType;
+import org.raml.model.Raml;
+
+final class Generators {
+    private static final Pattern PILLAR_AND_TIER_PATTERN = Pattern
+            .compile("(command/api|command/controller|command/handler|query/api|query/controller|query/view)");
```

Comment:
Events don't make any sense for the Rest adapter generator.  Do we want to move this into the framework as it is specific to the RAML to adapter generation.  Worth some further discussion.